### ISSUE TITLE
[NPC] Vyragosa's corpse traveled underground.

### ISSUE THREAD

**Issue by Arkaci** [WoWManiaUK/Redemption#6009]
After killing it, its body went under the snow.Although I kited during the attack, part of its body was underground.

![ss1](https://github.com/WoWManiaUK/Redemption/assets/135339611/93f55f2f-b857-471c-a669-216fc53cab7e)
![ss2](https://github.com/WoWManiaUK/Redemption/assets/135339611/a702c29d-04c3-42d0-bbfc-ef286384d471)

**Comment by GMZinius** [comment]
Confirmed exactly how you explained it to me in our ingame chat @Arkaci 

https://i.gyazo.com/ddfd97170aec5a91c00830619a09f97e.mp4


@Rushor 
As you see in the video above, When fighting She is clipping the terrain. 
Once Vyragosa is killed at coods 43. 21, 55.54 (Arkaci killed at 43. 21, 56.09) Vyragosa disapears like She died in a different phase or something. 

I did try /target Vyragosa on both phases, no target found
